### PR TITLE
NC-5

### PR DESCRIPTION
**Pull request by starikandrei**

### CODE REVIEW

File: blocklearning/lib/app/modules/home/views/home_view.dart [hunk 1]
```diff
@@ -1,13 +1,65 @@
...
+          itemBuilder: (context, index) {
+            return controller.pages[index];
+          },
+        ),
+        bottomNavigationBar: NavigationBar(
+          destinations: [
+            CustomNavigationButton(
+              icon: Icons.home,
+              label: LocaleKeys.main.tr,
+              isSelected: controller.selectedPageIndex.value == 0,
```
Comment (dadsparta) [inline blocklearning/lib/app/modules/home/views/home_view.dart:39]: May we move it to function with int parameters?

File: blocklearning/lib/core/theme/app_text_styles.dart [hunk 2]
```diff
@@ -3,22 +3,22 @@ import 'package:flutter/material.dart';
 
 abstract class AppTextStyles {
```
Comment (dadsparta) [inline blocklearning/lib/core/theme/app_text_styles.dart:4]: It is what we speak about with Stas when i create different text styles. It's should be made with the help of fabric constructor.

**Review by dadsparta — COMMENTED** [review]
Some questions before we merge it

**Review by dadsparta — APPROVED** [review]

File: blocklearning/lib/core/theme/app_text_styles.dart (hunk 2 above)
Comment (axpreparator) [reply blocklearning/lib/core/theme/app_text_styles.dart:4]: i think we don't need in so hard solution

File: blocklearning/pubspec.lock
Comment (axpreparator) [inline blocklearning/pubspec.lock:1]: pubspec.lock to git ignor

File: blocklearning/lib/generated/locales.g.dart
Comment (axpreparator) [inline blocklearning/lib/generated/locales.g.dart:1]: generated files to git ignor

**Review by axpreparator — APPROVED** [review]

**Pull request merged**